src/graphics/programlib/programlib_phong.js

```diff
@@ -192,7 +229,18 @@ pc.programlib.phong = {
         //////////////////////////////
         // GENERATE FRAGMENT SHADER //
         //////////////////////////////
+        var fshader;
         code = getSnippet(device, 'fs_precision');
+
+        if (options.customPS) {

```

Comment:
And if we would change PS, I would name it then Fragment instead of Pixel. vs/fs